tests/playwright/src/model/core/states.ts

```diff
@@ -1,5 +1,5 @@
 /**********************************************************************
- * Copyright (C) 2023 Red Hat, Inc.
+ * Copyright (C) 2023-2025 Red Hat, Inc.
```

Comment:
@odockal how do we handle copyright in our tests? should we do a general update of the copyrights?